pallets/gear/src/lib.rs

```diff
@@ -563,6 +662,8 @@ pub mod pallet {
                 <T as Config>::Currency::minimum_balance().unique_saturated_into();
 
             let mut max_gas_spent = 0;
+            let mut gas_to_send = 0;
```

Comment:
better to keep naming as in other places
```suggestion
            let mut reserved = 0;
```

Reply:
this can be calculated at the end of the processing only once